src/pages/UserPage/UserPage.tsx

```diff
@@ -124,6 +167,41 @@ const UserPage: FC = () => {
     [deactivationDate, userId, deactivateUser, t, name, setError]
   )
 
+  const onActivateSubmit: SubmitHandler<FormValues> = useCallback(
+    async (values) => {
+      const { user_activation_notification } = values
+
+      const roleIdsArray = roleOptions
```

Comment:
**[followup]**

As a followup to the previous comment, if you would use just the "value" for those options, there would be no need to filter or map anything here. "user_activation_notification" would already be an array of role id-s and could be passed directly to BE.